docker-compose.yml

```diff
@@ -170,9 +170,10 @@ services:
   node:
     image: ghcr.io/danskernesdigitalebibliotek/dpl-go-node:0.0.12
     labels:
-      lagoon.type: node
       provenance: false
-
+      lagoon.type: node-persistent
```

Comment:
Beware of this change. According to [the Lagoon docs](https://docs.lagoon.sh/concepts-advanced/service-types/): 

> Once a lagoon.type is defined and the environment is deployed, changing it to a different type is not supported and could result in a broken environment.

I am not sure whether it is a problem here.

Reply:
I believe it is only a problem if you introduce it in an existing environment - eg. pushing a commit with the change in an already deployed environment

Reply:
In production environments the service does not exist yet.